setup/MSBuild.swixproj

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <Import Project="$(MicroBuildDir)MicroBuild.Core.props" />
+  <Import Project="$(GitVersioningDir)dotnet\Nerdbank.GitVersioning.targets" />
+
+  <PropertyGroup>
+    <ProductVersion>3.10</ProductVersion>
+    <ProjectGuid>c0dfb7dd-3336-4f6c-bbd2-8c475f19d9bc</ProjectGuid>
+    <SchemaVersion>2.0</SchemaVersion>
+    <OutputArchitecture>neutral</OutputArchitecture>
+    <OutputLocalized>false</OutputLocalized>
+    <OutputName>MSBuild</OutputName>
+    <OutputType>vsix</OutputType>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
+    <IsPackage>true</IsPackage>    
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutDir>$(BaseOutputPath)setup\</OutDir>
+    <OutputPath>$(BaseOutputPath)setup\</OutputPath>
+    <X86BinPath>$(BaseOutputPath)X86\$(OS)\$(Configuration)\</X86BinPath>
+    <X64BinPath>$(BaseOutputPath)X64\$(OS)\$(Configuration)\</X64BinPath>
+  </PropertyGroup>
+
+  <!-- Needed for the MicroBuild signing plugin -->
+  <ItemGroup>
+    <SigningTarget Include="$(OutDir)$(OutputName).$(OutputType)"/>
+    <FilesToSign Include="@(SigningTarget)">
+      <Authenticode>Vsix</Authenticode>
+    </FilesToSign>
+  </ItemGroup>  
+
+  <ItemGroup>
+    <Package Include="Files.swr" />
+  </ItemGroup>
+
+  <Target Name="GetVxisPackageVersion" BeforeTargets="Build" DependsOnTargets="GetBuildVersion">
```

Comment:
Nit: can you be more specific than "build"? And maybe rename this to "SetVsixProperties"?